ice/server.py

```diff
@@ -0,0 +1,8 @@
+from uvicorn import main
+
+params_by_name = {p.name: p for p in main.params}
+params_by_name["app"].default = "ice.routes.app:app"
+params_by_name["port"].default = 8935
```

Comment:
It'd be nice to have reload=True when installing ice in editable mode. Can we detect that?

Alternatively, maybe for now we should always run with reload=True

Reply:
I think for the sake of exernal package users it's best not to touch the default of reload. Internally it's easy to add `--reload` in development.